torch/lib/TH/CMakeLists.txt

```diff
@@ -384,6 +384,7 @@ INSTALL(TARGETS TH
   ARCHIVE DESTINATION "${TH_INSTALL_LIB_SUBDIR}")
 
 INSTALL(FILES
+  generalFunc.h
```

Comment:
The naming of this file does not really fit with the rest of the library.
Also, given his content, is it necessary? If so maybe something like `THUtil.h` ?

Reply:
It's a good name. I will take it.